smart_proxy_dns_powerdns.gemspec

```diff
@@ -20,4 +20,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency('mocha')
 
   s.add_dependency('mysql2')
+  s.add_dependency('pg')
```

Comment:
The alternative is to leave both deps off, put them only in Gemfile and state that one or the other has to be installed as well for it to work.  I don't have any preference, it'll only require client libraries I suppose for both if you keep both here.

Optionally when packaging the plugin you could have extra subpackages for the respective backends if you wished to abstract it out.